plugins/opensource/table.md

```diff
@@ -672,7 +672,7 @@ tinymce.init({
 });
 ```
 
-> **Note**: The advanced tabs of the table, row, and cell properties dialogs use the `colorpicker` to allow for border and background colors to be applied. See docs to use and configure a custom [colorpicker]({{site.baseurl}}/plugins/colorpicker/).
+> **Note**: The advanced tabs of the table, row, and cell properties dialogs use the `colorpicker` to allow for border and background colors to be applied. See docs to use and configure a custom [colorpicker]({{site.baseurl}}/configure/content-appearance/#textcoloroptions).
```

Comment:
Is this the correct link?

Reply:
yes. Color picker was a plugin in v4, core in v5. I only recently removed the plugin page which only told you about this change.
V4: https://www.tiny.cloud/docs-4x/plugins/colorpicker/
V5: https://www.tiny.cloud/docs/configure/content-appearance/#textcoloroptions

Note: I now notice that "text color options" is a misnomer. These options are for all instances of the color picker, not just those related to text.